code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -171,23 +191,35 @@
 		if(ALIEN_BODYPART,LARVA_BODYPART) //aliens take double burn //nothing can burn with so much snowflake code around
 			burn *= 2
 
-	var/can_inflict = max_damage - get_damage()
-	if(can_inflict <= 0)
-		return FALSE
+	var/wounding_type = (brute > burn ? WOUND_BRUTE : WOUND_BURN)
+	var/wounding_dmg = max(brute, burn)
+	if(wounding_type == WOUND_BRUTE && sharpness)
+		wounding_type = WOUND_SHARP
+	// i know this is effectively the same check as above but i don't know if those can null the damage by rounding and want to be safe
+	if(owner && wounding_dmg > 4 && wound_bonus != CANT_WOUND)
+		// if you want to make tox wounds or some other type, this will need to be expanded and made more modular
+		// handle all our wounding stuff
+		check_wounding(wounding_type, wounding_dmg, wound_bonus, bare_wound_bonus)
```

Comment:
in your design image, you said that damage on a limb (and not just the existing wounds on a limb, as that was a separate point on the list) should affect its wound rolls, but I don't see anything that implements that concept in this PR. 

did you go back on that design choice, or did you just forget to add it?